builder/vsphere/clone/step_customize.go

```diff
@@ -202,24 +202,24 @@ func (c *CustomizeConfig) Prepare() ([]string, []error) {
 		errs = append(errs, fmt.Errorf("one or more `network_interface` must be provided"))
 	}
 
-	options_number := 0
+	optionsNumber := 0
 	if c.LinuxOptions != nil {
-		options_number = options_number + 1
+		optionsNumber = optionsNumber + 1
```

Comment:
May I suggest using the shorter increment syntax here?
```suggestion
		optionsNumber++
```

That would apply to the other increments too

Reply:
Good catch! Resolved in c5db4fe.